### ISSUE TITLE
Add linting scripts for use in CI

### ISSUE THREAD

**Issue by caesarsol** [accurat/accurapp#2]
emulate this:

```
"scripts": {
    "lint": "node node_modules/eslint/bin/eslint.js src/",
    "lint:errors": "node node_modules/eslint/bin/eslint.js --quiet src/"
}
```

**Comment by markov00** [comment]
- [ ] Add lint only at pre-commit

- [ ] Add lint only on diff files

**Comment by marcofugaro** [comment]
pre-commit example

https://github.com/vuejs/vue/blob/v2.2.2/build/git-hooks/pre-commit

**Comment by marcofugaro** [comment]
Added lint command in 8b82128 which lints only changed files in src/ folder.

Sould we also add the (removable) hooks on pre-commit which prevents commiting if there are any warnings? Using [husky](https://github.com/typicode/husky)

What do you say @accurat/developers?

**Comment by marcofugaro** [comment]
Closed in #40 .

Linting in pre-commit seems too strict. If someone wishes to, he can enable it per project using [husky](https://github.com/typicode/husky)
